extension/js/common/composer/interfaces/composer-types.ts

```diff
@@ -61,3 +61,6 @@ export type ComposerUrlParams = {
 };
 
 export type PubkeyResult = { pubkey: string, email: string, isMine: boolean };
+
+export type ComposerPopoverItem = { type: 'selectEncryptionType', HTMLContent: string, data: EncryptionType, iconPath?: string };
```

Comment:
It seems the `type: 'selectEncryptionType'` is not currently used. It's best to not introduce code "in case it's useful in the future" because who knows, the future may not be what we expect :) Unless I overlooked some usage of this.

Reply:
Thanks, I added it because I thought it's useful in the future but I agree that the future may not be what we expect  :)

I will remove it